kong/client.go

```diff
@@ -37,7 +38,9 @@ var (
 // Kong cluster
 type Client struct {
 	client                  *http.Client
-	baseURL                 string
+	defaultRootURL          string
+	workspace               string       // Do not access directly. Use Get/SetWorkspace().
```

Comment:
note: this comment needs to be updated when we get rid of the `GetWorkspace()` name